src/display/components/help_text.rs

```diff
@@ -20,34 +21,31 @@ const TEXT_TAB_TIP: &str = " Use <TAB> to rearrange tables.";
 
 impl HelpText {
     pub fn render(&self, frame: &mut Frame<impl Backend>, rect: Rect) {
-        let text = {
-            let pause_content = if self.paused {
-                TEXT_WHEN_PAUSED
-            } else {
-                TEXT_WHEN_NOT_PAUSED
-            };
-
-            let dns_content = if rect.width <= FIRST_WIDTH_BREAKPOINT {
-                ""
-            } else if self.show_dns {
-                TEXT_WHEN_DNS_SHOWN
-            } else {
-                TEXT_WHEN_DNS_NOT_SHOWN
-            };
+        let pause_content = if self.paused {
+            TEXT_WHEN_PAUSED
+        } else {
+            TEXT_WHEN_NOT_PAUSED
+        };
 
-            let tab_text = if rect.width <= SECOND_WIDTH_BREAKPOINT {
-                ""
-            } else {
-                TEXT_TAB_TIP
-            };
+        let dns_content = if rect.width <= FIRST_WIDTH_BREAKPOINT {
+            ""
+        } else if self.show_dns {
+            TEXT_WHEN_DNS_SHOWN
+        } else {
+            TEXT_WHEN_DNS_NOT_SHOWN
+        };
 
-            [Text::styled(
-                format!("{}{}{}", pause_content, tab_text, dns_content),
-                Style::default().modifier(Modifier::BOLD),
-            )]
+        let tab_text = if rect.width <= SECOND_WIDTH_BREAKPOINT {
+            ""
+        } else {
+            TEXT_TAB_TIP
         };
-        Paragraph::new(text.iter())
-            .alignment(Alignment::Left)
-            .render(frame, rect);
+
+        let text = Span::styled(
+            format!("{}{}{}", pause_content, tab_text, dns_content),
```

Comment:
Sticking strings together in Rust always gives me a bit of grief, but I think this also works here:
```rust
[pause_content, tab_text, dns_content].concat(),
```
To me it seems a tiny bit cleaner and is, apparently, [about 3x faster](https://github.com/hoodie/concatenation_benchmarks-rs) because it can pre-compute the size of the final string I think. We are talking nanoseconds though, so it really doesn't matter and is a matter of style I think :)

Reply:
Aha, good to know - thanks! I made the change.

Reply:
Btw, @TheLostLambda - I sent you an email on a different subject. Mentioning it here because I remember there were some spam-filtering issues between us in the past. :)